config/application.rb

```diff
@@ -14,17 +14,6 @@
 
 module Upaya
   class Application < Rails::Application
-    config.generators do |g|
-      g.test_framework :rspec,
-                       fixtures: true,
-                       view_specs: false,
-                       helper_specs: false,
-                       routing_specs: false,
-                       controller_specs: false,
-                       request_specs: false
-      g.fixture_replacement :factory_girl, dir: 'spec/factories'
-    end
-
```

Comment:
This is used for scaffolding a new app. For example, when you want to create a brand new Rails app, you can generate a bunch of boilerplate files by using the `scaffold` command, such as `rails generate scaffold User name:string`, which will generate the migrations, views, controllers, and tests. You can customize what gets generated with code like what we had, but we never actually used it because we didn't generate the app from scratch using scaffolding. Anyone forking this repo for their own purposes wouldn't use this either. 

More info about generators here: http://guides.rubyonrails.org/generators.html